source/scripts/app.js

```diff
@@ -503,9 +503,19 @@ function homeSearchFunction() {
       e.preventDefault();
       // store search string and navigate to explore page
       // let searchQuery = e.target.value;
+      let searchQuery = e.target.value;
```

Comment:
This needs to be an object in order for `populateExplorePage` to work correctly
```suggestion
      let searchQuery = { query: e.target.value };
```